dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/MetadataBitstreamRestRepository.java

```diff
@@ -109,12 +113,16 @@ public Page<MetadataBitstreamWrapperRest> findByHandle(@Parameter(value = "handl
                         List<PreviewContent> prContents = previewContentService.hasPreview(context, bitstream);
                         // Generate new content if we didn't find any
                         if (prContents.isEmpty()) {
-                            fileInfos = previewContentService.getFilePreviewContent(context, bitstream);
-                            // Do not store HTML content in the database because it could be longer than the limit
-                            // of the database column
-                            if (!StringUtils.equals("text/html", bitstream.getFormat(context).getMIMEType())) {
-                                for (FileInfo fi : fileInfos) {
-                                    previewContentService.createPreviewContent(context, bitstream, fi);
+                            boolean allowComposePreviewContent = configurationService.getBooleanProperty
+                                    ("allow.compose.file-preview", false);
```

Comment:
Rename it to make sure this property disable creating a file preview on item page load. COnsider the name:
`create.file-preview.on-item-page-load`